modules/library/referencing/src/main/java/org/geotools/referencing/factory/ThreadedAuthorityFactory.java

```diff
@@ -665,7 +665,11 @@ public synchronized CoordinateReferenceSystem createCoordinateReferenceSystem(fi
             crs = (CoordinateReferenceSystem) cached;
         } else {
             crs = getBackingStore().createCoordinateReferenceSystem(code);
+            if (LOGGER.isLoggable(Level.FINER)) {
```

Comment:
Why checked logging here, and usage of a lambda a line below? Both legit, just trying to figure out the logic.

Reply:
I don't know. Just started to use lambda.